docroot/themes/humsci/humsci_basic/src/scss/ckeditor/imports.scss

```diff
@@ -264,15 +264,24 @@ $px-only: true;
   blockquote:nth-child(n) {
     @include hb-blockquote;
     overflow: visible;
-
-    font-size: 24px;
+    font-size: 24px !important;
+    line-height:35.56px !important;
```

Comment:
Always try to use relative dimensions (`LINE_HEIGHT_PX/FONT_SIZE`) for `line-height`, to allow it to automatically adjust to font size. In this case `35.56/28 = 1.27`